### ISSUE TITLE
Feature Request: ability to offset page number for pdfs including (for example) cover pages before page 1

### ISSUE THREAD

**Issue by joelstrouts** [burtonator/polar-bookshelf#1564]
For documents without hyperlinked contents pages, you need to actually use the page numbers in the contents - and if the book included cover pages then "go to page 52" does not take you to the 52nd numbered page in the book because of the offset.

Feature request: ability to set an offset manually when this occurs

**Comment by burtonator** [comment]
Yes... this is a good idea and I'll try to get it on the agenda.
